Source/VerveAdapter.swift

```diff
@@ -68,13 +68,14 @@ final class VerveAdapter: PartnerAdapter {
     /// - parameter request: Information about the ad load request.
     /// - parameter completion: Closure to be performed with the fetched info.
     func fetchBidderInformation(request: PreBidRequest, completion: @escaping ([String : String]?) -> Void) {
-        guard let appToken else {
```

Comment:
if we aren't using the appToken instance property anymore, can we remove it?

Reply:
`setUp` is using it.

```
        guard let appToken = configuration.credentials[APP_TOKEN_KEY] as? String else {
            let error = error(.initializationFailureInvalidCredentials, description: "The app token was invalid")
            log(.setUpFailed(error))
            completion(error)
            return
        }
        self.appToken = appToken

        HyBid.initWithAppToken(appToken) { success in
            if success {
                self.log(.setUpSucceded)
                completion(nil)
            } else {
                let error = self.error(.initializationFailureUnknown)
                self.log(.setUpFailed(error))
                completion(error)
            }
        }
```

Reply:
Oh I see what you actually mean.  One sec.